cmake/protobuf-config.cmake.in

```diff
@@ -1,19 +1,30 @@
 # Version info variables
-set(PROTOBUF_VERSION        "@protobuf_VERSION@")
-set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(Protobuf_VERSION        "@protobuf_VERSION@")
+set(Protobuf_VERSION_STRING "@protobuf_VERSION_STRING@")
 
 # Imported targets
 include("${CMAKE_CURRENT_LIST_DIR}/protobuf-targets.cmake")
 
-# Compute the installation prefix relative to this file.
-get_filename_component(_PROTOBUF_IMPORT_PREFIX
-  "${_PROTOBUF_PACKAGE_PREFIX}" PATH)
-get_filename_component(_PROTOBUF_IMPORT_PREFIX
-  "${_PROTOBUF_IMPORT_PREFIX}" PATH)
-get_filename_component(_PROTOBUF_IMPORT_PREFIX
-  "${_PROTOBUF_IMPORT_PREFIX}" PATH)
+if(NOT DEFINED protobuf_TARGETS_ALIASES OR "${protobuf_TARGETS_ALIASES}")
```

Comment:
Again, this is counter to standard cmake practices. Import targets should ALWAYS have a namespace. See [CMP0028](https://cmake.org/cmake/help/v3.5/policy/CMP0028.html)